sorting/odd_even_sort.cpp

```diff
@@ -1,53 +1,66 @@
-/* C++ implementation Odd Even Sort */
```

Comment:
There should be a header comment check other files or the contribution guidelines